public/yoga.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="../assets/css/yoga.css" />
+    <title>Pranayam-Yoga</title>
+  </head>
+  <body>
+    <h1>Pranayam-Yoga</h1>
+
+    <div class="container" id="container">
+      <div class="circle"></div>
+
+      <p id="text"></p>
+
+      <div class="pointer-container">
+        <span class="pointer"></span>
+      </div>
+
+      <div class="gradient-circle"></div>
+    </div>
+    <!-- <audio src="../assets/sounds/yoga-beat.mp3" controls></audio> -->
+    <embed name="yoga-beat" src="../assets/sounds/yoga-beat.mp3" width="150" height="80" loop="true" autostart="false">
```

Comment:
Don't use embed,
Use audio tag as it's audio it's playing 
![image](https://user-images.githubusercontent.com/38184186/112936639-cd695200-9143-11eb-9223-9ab9b612e7de.png)

Reply:
Thank you sir for your valuable feedback, I have changed the code accordingly.